src/Spectre.libGenetic/FitnessFunction.h

```diff
@@ -33,7 +34,7 @@ class FitnessFunction
     /// </summary>
     /// <param name="individual">The individual.</param>
     /// <returns>Non-negative score, which is greater for more optimal individuals.</returns>
-    virtual ScoreType operator()(const Individual &individual) = 0;
+    virtual ScoreType computeFitness(const Individual &individual) = 0;
```

Comment:
I'm not sure, why this has been renamed. Could you explain?

Reply:
I had some errors with operator... so instead of dealing with a problem I just went around it... :D